src/app/rpmostree-builtin-status.c

```diff
@@ -480,6 +480,11 @@ print_one_deployment (RPMOSTreeSysroot *sysroot_proxy,
       g_autofree char *canonrefspec = rpmostree_refspec_to_string (refspectype, refspec_data);
       switch (refspectype)
         {
+        case RPMOSTREE_REFSPEC_TYPE_COMMIT:
+          {
+            g_print ("Commit: %s", refspec_data);
```

Comment:
We actually don't in this PR, because that case checked for `TYPE_OSTREE` (to exclude rojig).

Reply:
Ahh right. Hmm, I think I still prefer the consistency of `ostree://<sha256>`. E.g. it maintains the nice property that it can fed directly to `rpm-ostree rebase`, just like the other canonical refspec types.